cfme/v2v/migrations.py

```diff
@@ -363,17 +371,51 @@ def plan_in_progress(self, plan_name):
             return False
 
 
+class MigrationDashboardView59z(MigrationDashboardView):
+    infra_mapping_list = InfraMappingList("infra-mappings-list-view")
+
+    @property
+    def is_displayed(self):
+        return (self.navigation.currently_selected == ['Compute', 'Migration'] and
+            (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0) and
+            (len(self.browser.elements('.//div[contains(@class,"card-pf")]')) > 0))
+
+
+class InfrastructureMappingView(BaseLoggedInPage):
+    infra_mapping_list = InfraMappingList("infra-mappings-list-view")
+    create_infrastructure_mapping = Text(locator='(//a|//button)'
+                                                 '[text()="Create Infrastructure Mapping"]')
+    paginator_view = View.include(InfrastructureMappingsPaginatorPane, use_parent=True)
+    search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+    clear_filters = Text(".//a[text()='Clear All Filters']")
+    # Used for Ascending/Descending sort
+    sort_order = Text(".//button[./span[contains(@class,'sort-direction')]]")
+    # Used to select filter_by 'Name' or 'Description'
+    filter_by_dropdown = SelectorDropdown('id', 'filterFieldTypeMenu')
+    # USed to select sort by options like 'Name', 'Number of Associated Plans'
+    sort_by_dropdown = SelectorDropdown('id', 'sortTypeMenu')
+
+    @property
+    def is_displayed(self):
+        return (self.navigation.currently_selected ==
+            # TODO: Remove nxt line & uncomment line below that, once /manageiq-v2v/issues/726 fixed
+            ['Compute', 'Migration'] and
+            # ['Compute', 'Migration', 'Infrastructure Mappings'] and
+            len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0 and
+            (self.create_infrastructure_mapping.is_displayed or
+                self.infra_mapping_list.is_displayed))
+
+
 class AddInfrastructureMappingView(View):
     form = InfraMappingWizard()
+    form_title_text = VersionPicker({
```

Comment:
This was failing to `.pick()` right version/item while it was inside `is_displayed` so moved it out, now it works as expected.